.gitignore

```diff
@@ -9,7 +9,6 @@ config/local.json
 var
 build/config.json
 core/build/config.json
-core/build/cache-version.json
```

Comment:
This file should stay in ignore - it will be then created at `clone` but ignored with the subsequent commits :)